polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md

```diff
@@ -1027,97 +2507,14 @@ After migrating use the following RegExp to check for any additional instances o
 | `--p-space-32`               | `--p-space-3200`  |
 
-### Recommended token migration workflow
-
-When running token migrations we suggest the following workflow:
-
-- Handle automated migrations
-  ```sh
-  # Example migration
-  npx @shopify/polaris-migrator ...
-  # Stage all migrated files
-  git add .
-  # Format staged files only
-  git diff --staged --name-only | xargs npx prettier --write
-  # Commit automated migrations
-  git commit -m "Migrate X custom properties from Polaris v11 to v12"
-  ```
-- Handle manual migrations
-  - Search for token RegExps and handle manual migrations
-    <br />
-
-```sh
-# Stage all manually migrated files
-git add .
-# Format staged files only
-git diff --staged --name-only | xargs npx prettier --write
-# Commit manual migrations
-git commit -m "Manually migrate X custom properties from Polaris v11 to v12"
-```
-
-- Optionally if you use `stylelint-polaris`, you can check for errors after all custom property migrations are finished
-  ```sh
-  npx stylelint <path>
-  ```
-
-### `@shopify/polaris-tokens` updates
-
-#### Renames
-
-- `getCustomPropertyNames` renamed to `getThemeVarNames`
-- `createVar` renamed to `createVarName`
-
-#### Deprecations
-
-##### Deprecated Utilities
-
-If you are using these utilities, feel free to copy them from v11 into your own codebase.
-
-- `createExact`
-- `createMetadata`
-- `getKeyframeNames`
-- `getUnit`
-- `isKeyOf`
-- `rem`
-- `removeMetadata`
-- `toEm`
-- `tokensToRems`
-
-##### Deprecated Types
-
-- `BreakpointsAliasDirectionMediaConditions`
-- `BreakpointsMediaConditions`
-- `MetaBreakpointsTokenGroup`
-- `Tokens` (replaced by `Theme`)
-
-##### Deprecated all JSON exports
-
-- `@shopify/polaris-tokens/json/border.json`
-- `@shopify/polaris-tokens/json/breakpoints.json`
-- `@shopify/polaris-tokens/json/color.json`
-- `@shopify/polaris-tokens/json/font.json`
-- `@shopify/polaris-tokens/json/height.json`
-- `@shopify/polaris-tokens/json/motion.json`
-- `@shopify/polaris-tokens/json/shadow.json`
-- `@shopify/polaris-tokens/json/space.json`
-- `@shopify/polaris-tokens/json/text.json`
-- `@shopify/polaris-tokens/json/width.json`
-- `@shopify/polaris-tokens/json/zIndex.json`
-
-If you are using these exports, update the implementation to import `themes` and `JSON.stringify` the theme you need.
-
-```diff
-- const color = require('@shopify/polaris-tokens/json/color.json');
-+ const {themes} = require('@shopify/polaris-tokens');
-+ const color = JSON.stringify(themes.light.color);
-```
+</CollapsibleDetails>
 
 ## Manual updates and fixes
 
 ### A new web font
 
 The new design language comes with a web font called [Inter via Google Fonts](https://fonts.google.com/specimen/Inter).
-Polaris references this font but doesn't load it. Apps will need to load the font, otherwise it will fallback to to the user's system font.
+Polaris references this font but doesn't load it. Apps will need to load the font, otherwise it will fallback to the user's system font.
```

Comment:
```suggestion
Polaris references this font but does not load it. Your app will need to load the font, otherwise it will fallback to the user's system font.
```